pkg/kubelet/dockertools/docker.go

```diff
@@ -64,10 +64,10 @@ type DockerInterface interface {
 	StartContainer(id string) error
 	StopContainer(id string, timeout int) error
 	RemoveContainer(id string, opts dockertypes.ContainerRemoveOptions) error
-	InspectImage(image string) (*docker.Image, error)
-	ListImages(opts docker.ListImagesOptions) ([]docker.APIImages, error)
-	PullImage(opts docker.PullImageOptions, auth docker.AuthConfiguration) error
-	RemoveImage(image string) error
+	InspectImage(image string) (*dockertypes.ImageInspect, error)
+	ListImages(opts dockertypes.ImageListOptions) ([]dockertypes.Image, error)
+	PullImage(image string, auth dockertypes.AuthConfig, opts dockertypes.ImagePullOptions) error
```

Comment:
@MHBauer We do this, because:
1) Engine-api has moved image out of `ImagePullOptions` https://github.com/docker/engine-api/pull/162, it will be easier to bump up the engine-api version later.
2) This is more intuitive https://github.com/docker/engine-api/issues/137. :)